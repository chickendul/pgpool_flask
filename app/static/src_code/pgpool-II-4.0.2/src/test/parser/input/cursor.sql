version https://git-lfs.github.com/spec/v1
oid sha256:9226d958d5552ad5ad7ae3a020b3231a0f65d9a18858b2a96ed93308a2ed8028
size 912
