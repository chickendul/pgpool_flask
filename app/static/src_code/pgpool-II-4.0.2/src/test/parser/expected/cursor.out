version https://git-lfs.github.com/spec/v1
oid sha256:0f4bf89c0cc22fc9f54ef78ce025dc2b542bb91bd57280d6beb93298109d17e8
size 1024
